tests/units/offline/test_lidar.py

```diff
@@ -0,0 +1,45 @@
+import grpc
+import pytest
+from google.protobuf.wrappers_pb2 import BoolValue, FloatValue
+from reachy2_sdk_api.mobile_base_lidar_pb2 import (
+    LidarObstacleDetectionEnum,
+    LidarObstacleDetectionStatus,
+    LidarSafety,
+)
+
+from reachy2_sdk.subparts.lidar import Lidar
+
```

Comment:
Thanks for the unit testing :)
I've output the coverage log to see if something is missing. The following functions are not tested:
reset_safety_default_values()
__repr__ -> test the output string